content/sensu-enterprise/2.6/integrations/opsgenie.md

```diff
@@ -8,120 +8,12 @@ menu:
   sensu-enterprise-2.6:
     parent: integrations
 ---
-**ENTERPRISE: Built-in integrations are available for [Sensu Enterprise][1]
-users only.**
 
-- [Overview](#overview)
-- [Configuration](#configuration)
-  - [Example(s)](#examples)
-  - [Integration Specification](#integration-specification)
-    - [`opsgenie` attributes](#opsgenie-attributes)
+_WARNING: To use the OpsGenie integration, please [upgrade to the latest version of Sensu Enterprise][1]._
 
-## Overview
+Following the shutdown of the OpsGenie v1 API on June 30, 2018, Sensu Enterprise 2.6 no longer supports the OpsGenie integration.
+To use the Sensu Enterprise OpsGenie integration, please [upgrade to the latest version of Sensu Enterprise][1].
+See the [Sensu Enterprise 3.0 upgrade guide][1] and the [OpsGenie integration docs][2] for more information.
```

Comment:
As this points to `latest`, `Sensu Enterprise 3.0 upgrade guide` won't be true when a new major or minor version of Sensu Enterprise comes out. The 3.0 docs have the only upgrade guide that is relevant to the OpsGenie change currently. Or is the upgrade guide going to brought up to all versions 3.x and above? That would make `latest` continue to work (still would want to remove the 3.0 label I think) and cover future scenarios where users may be upgraded from 2.x to say 3.1.

That all said, the Enterprise user base is much smaller than Core, so maybe this isn't a large issue to begin with. :-).

Reply:
Updated all these links to point to 3.0 specifically. This emphasizes that "3.0 or later" is the key bit here and maintains consistency. In the future, the "You're viewing documentation for an older or prerelease version..." bar will tip off users that a newer version exists.